draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -639,6 +665,19 @@ The wire format for each parameter is a sequence of IP addresses in network
 byte order.  Like an A or AAAA RRSet, the list of addresses represents an
 unordered collection.
 
+These parameters MAY be repeated multiple times within a record.
+When this happens, the list of addresses SHOULD be concatenated.
+
+When selecting between AAAA and A records to use, clients may use an
+approach such as {{!HappyEyeballsV2=RFC8305}}.  Within an address
+family, clients SHOULD pick addresses to use in a random order.
+
+When only "a" parameters are present, IPv6-only clients MUST either
+resolve an AAAA records from the SvcDomainName or attempt
```

Comment:
I think this requirement already exists, regardless of whether the client is IPv6-only.